src/jit/morph.cpp

```diff
@@ -2859,11 +2846,13 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* call)
     // On remorph, we grab it from the arg table.
     unsigned numArgs = 0;
 
+    // TODO: this comment is no longer correct, do an approptiate fix.
```

Comment:
I want to fix this in a separate PR, because now we can morph `call` and `args` in the general postorder. It will allow us to optimize `call` node better and will introduce some asm diffs that I do not want to include in this PR.

Reply:
Typo "appropriate"